### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Goles**
1. overridding should be overriding.

### CODE REVIEW

File: docs/OptimizationTips.rst [hunk 1]
```diff
@@ -211,7 +211,7 @@ bridged to NSArray, use ContiguousArray instead of Array:
   class C { ... }
   var a: ContiguousArray<C> = [C(...), C(...), ..., C(...)]
 
-Advice: Use inplace mutation instead of object-reassignment
```
Comment (krememek) [inline docs/OptimizationTips.rst]: 'inplace' is a term.  The other changes look fine.

**Comment by adityatrivedi** [comment]
IMO it would be better to squash the commits into one single commit.

**Comment by krememek** [comment]
Yes please squash into a single commit.

**Comment by Goles** [comment]
@krememek sorry for not doing it earlier, took a while to pull the whole repo and Github didn't allow me to squash :+1:

**Pull request merged**